CHANGELOG.rst

```diff
@@ -1,6 +1,22 @@
 `Unreleased <https://github.com/pace-neutrons/Euphonic/compare/v0.6.4...HEAD>`_
 ----------
 
+- Changes:
+
+  - The following deprecated features have been removed:
+
+    - The ``return_mode_widths`` argument in ``calculate_qpoint_phonon_modes/frequencies``
```

Comment:
Should this be `ForceConstants.calculate_qpoint_phonon_modes() and ForceConstants.calculate_qpoint_frequencies()`? It's not obvious what functionality is affected.